packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -1184,6 +1184,7 @@ const messages = {
           unShare: 'Avslutt deling',
           shareLink: 'Kopier lenke',
         },
+        previewInformation: 'Forhåndsvisning av mappe. Den er ikke tilgjenglig for andre før du setter mappa som delt.',
```

Comment:
I mitt hode er dette bedre ordlyd.
```suggestion
        previewInformation: 'Forhåndsvisning av delt mappe. Mappa vil ikke bli tilgjenglig for andre før du setter den som delt.',
```

Reply:
Jeg er egentlig enig. Jeg har oppdatert ordlydene i alle 5 for å matche sånn ca (fordel om oversettelsene sier omtrentlig det samme)